host-ext/crouton/background.js

```diff
@@ -589,13 +643,14 @@ function websocketClose() {
     checkUpdate(false);
 }
 
-/* Called when window in focus changes: feeback to the extension so the
+/* Called when window/tab in focus changes: feeback to the extension so the
```

Comment:
In my defense I inherited that one, and actually fixed it once before resetting the file accidentally :P